### PR TITLE
[NTOS:KE] KiIdleLoop(): Add DECLSPEC_NORETURN to it and its callers

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Be explicit about code flow.

## Proposed changes

Add DECLSPEC_NORETURN, Remove FASTCALL.

### CODE REVIEW

**Review by tkreuzer — APPROVED** [review]

**Comment by HBelusca** [comment]
@SergeGautherie Please fix the conflicts.

**Comment by SergeGautherie** [comment]
Rebased.

File: ntoskrnl/include/internal/ke.h [hunk 1]
```diff
@@ -740,6 +740,7 @@ VOID
 NTAPI
 KiInitializeBugCheck(VOID);
 
+// TODO: Add DECLSPEC_NORETURN, after checking it applies everywhere.
```
Comment (Extravert-ir) [inline ntoskrnl/include/internal/ke.h:743]: Please, finish the work

**Review by Extravert-ir — CHANGES_REQUESTED** [review]

File: ntoskrnl/include/internal/ke.h (hunk 1 above)
Comment (SergeGautherie) [reply ntoskrnl/include/internal/ke.h]: Finished.

**Comment by Extravert-ir** [comment]
I don't like powerpc changes, just remove them

**Comment by SergeGautherie** [comment]
> powerpc changes

Removed.

**Review by Extravert-ir — APPROVED** [review]

**Pull request merged**